ui/src/timeMachine/components/FunctionSelector.tsx

```diff
@@ -63,6 +82,11 @@ class FunctionSelector extends PureComponent<Props, State> {
     )
   }
 
+  private get period(): string {
+    const {aggregateWindow} = this.props
+    return aggregateWindow.period || AGG_WINDOW_AUTO
```

Comment:
Can the `aggregateWindow` be `null`, in which case this will throw an error? 

Might be the case for existing `builderConfig`s people have.